files/image_config/hostcfgd/hostcfgd

```diff
@@ -339,6 +339,8 @@ class HostConfigDaemon:
                     syslog.syslog(syslog.LOG_ERR, "{} - failed: return code - {}, output:\n{}"
                               .format(err.cmd, err.returncode, err.output))
             syslog.syslog(syslog.LOG_INFO, "Feature '{}' is stopped and disabled".format(key))
+        else:
+            syslog.syslog(syslog.LOG_ERR, "Unexpected status value for {}".format(key))
```

Comment:
Please also add the unexpected value you received to the log message.

Reply:
added